homeassistant/components/switchbot_cloud/sensor.py

```diff
@@ -43,6 +51,33 @@
     ),
 )
 
+METER_PRO_CO2_SENSOR_DESCRIPTIONS = (
```

Comment:
Let's not duplicate this list, since it's the previous one + CO2

Reply:
Updated